db/db_utils.go

```diff
@@ -373,6 +373,27 @@ func Aggregate(collection string, pipeline interface{}, out interface{}) error {
 	return errors.WithStack(pipe.All(out))
 }
 
+// AggregateContext runs an aggregation pipeline on a collection and unmarshals
+// the results to the given "out" interface (usually a pointer
+// to an array of structs/bson.M)
+func AggregateContext(ctx context.Context, collection string, pipeline interface{}, out interface{}) error {
+	session, db, err := GetGlobalSessionFactory().GetContextSession(ctx)
+	if err != nil {
+		err = errors.Wrap(err, "establishing db connection")
+		grip.Error(err)
+		return err
+	}
+	defer session.Close()
+
+	// NOTE: with the legacy driver, this function unset the
+	// socket timeout, which isn't really an option here. (other
+	// operations had a 90s timeout, which is no longer specified)
```

Comment:
I know it was just copied from the function above, but is this comment still relevant? It seems like a historical artifact of code that's no longer present and probably doesn't have any meaning to anyone anymore, so we could just delete both comments.